src/helpers/textInitials.js

```diff
@@ -0,0 +1,10 @@
+export const initialText = {
+  id: 'asdasasd',
```

Comment:
Not familiar to where are we using this yet but can id be changed to something more descriptive?